src/OpenLoco/Windows/Cheats.cpp

```diff
@@ -18,6 +18,11 @@
 
 using OpenLoco::GameCommands::CheatCommand;
 
+namespace OpenLoco::Ui::Windows::BuildVehicle
+{
+    extern bool displayLockedVehicles;
```

Comment:
I feel this should be the other way around. The cheats should own the bool and the window should ask the cheats what cheat is set. Eventually we will have more cheats that effect state and they should all be grouped within one cheat state structure.

Could you swap this around.

Reply:
done